src/common/constants.ts

```diff
@@ -1,6 +1,7 @@
-const VALID_DAYS = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday'];
+const VALID_DAYS = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'];
 
 const VALID_TIMES = [
+  9,
```

Comment:
This is something I would need to bring up during pod meeting, since I believe we are focusing on in-person, on-campus meet ups so timezone is not a current priority. As for your concern with manipulating the floats, the only operations I can think of at the moment we'd perform on them are equality, less/greater than, and intersection so I think we're on the safe side for now. I can revisit this in another PR if we start getting float issues and do the int conversion you suggest.

Reply:
Even on campus, people keep weird hours. I always personally lean towards leaving users as unrestricted as possible unless there is a very good reason.